src/hera/workflow.py

```diff
@@ -70,6 +74,7 @@ def __init__(
         image_pull_secrets: Optional[List[str]] = None,
         workflow_template_ref: Optional[str] = None,
         ttl_strategy: Optional[TTLStrategy] = None,
+        volume_claim_gc_strategy: Optional[VolumeClaimGCStrategy] = VolumeClaimGCStrategy.OnWorkflowCompletion,
```

Comment:
I am not sure this is a good assumption 🤔  maybe some users may want to keep the volumes around post workflow completion. The same logic was applied when the other optional parameters were set to `None`. There's no way to know whether the user does want a TTL strategy, so Hera offers the option to add it but makes no assumptions about it, for now at least. Any chance we can set this to `None` as well? I think we are ready to merge right after that into 3.2.0 🚀

Reply:
You're right! I would leave the default value as None.

Reply:
https://github.com/argoproj-labs/hera-workflows/pull/147/commits/ed486defdddd891d13b69244ca3e445c49e05161